docs/guides/cloud/branch-review.mdx

```diff
@@ -0,0 +1,123 @@
+  alt="Get to Branch Review"
+/>
+
+## Branch Details
+
+### View Options
+
+Once the GitHub integration is configured for a project, "Branch Review" is the default view of the **Branches** tab. Otherwise, a list of "Runs" filtered by this branch will default with tips for how to enable and maximize this feature.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-views.jpg"
+  alt="Displaying different views for branches"
+/>
+
+### Review Header
+
+At the left side of the header is the Git commit message, PR selector, PR status and base and target branch indicators. If there are multiple pull requests open for the same branch, you can select the pull request you want to review from the dropdown to the right of the commit message.
+
+The right side of the header includes the Cypress Cloud test run number and test summary. Clicking the test run number will link you directly to the [test overview](/guides/cloud/recorded-runs#Overview-tab).
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-header.jpg"
+  alt="Branch Review Header"
+/>
+
+### Review Test Status
+
+When on the review screen, you will see tabs Failures, Flaky, Pending, Added, and Modified. Each tab will show you the tests that fall into that category. You can click on a test to view the test details.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-details.jpg"
+  alt="Branch Review Details"
+  caption="Viewing new failing tests for this PR"
+/>
+
+:::info
+
+The arrows next to the numbers in each of the tabs indicate:
+
+- <svg
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M12 10V4M12 4L6 4M12 4L4 12"
+      stroke="#C62B49"
+      stroke-width="2"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+    />
+  </svg> total number increased in your branch
+- <svg
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M12 6V12M12 12L6 12M12 12L4 4"
+      stroke="#C62B49"
+      stroke-width="2"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+    />
+  </svg> total number decreased in your branch
+- <svg
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M9 12L13 8M13 8L9 4M13 8L3 8"
+      stroke="#C62B49"
+      stroke-width="2"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+    />
+  </svg> there are new items, but the total count remained the same - for example, <em>
+    1 resolved and 1 added
+  </em>
+
+:::
+
+### Review Test Comparison
+
+Navigating into the test detail view reveals a side-by-side comparison of the test in each branch which can help elevate the source of flake within tests throughout different attempts. Review the test definition panel for diff snapshots to help quickly determine changes in test code.
```

Comment:
"elevate the source of flake" - this is part of it, but perhaps more importantly it shows you the test results on both branches as well as artifacts so you can compare before vs after your PR